services/organization.js

```diff
@@ -282,6 +283,21 @@ class Organization {
 
     return { ...newOrg };
   }
+
+  /**
+  * Stores approved studies in the organization's collection.
+  *
+  * @param {string} studyName - The name of the study.
+  * @param {string} studyAbbreviation - The abbreviation of the study.
+  * @returns {void}
+  */
+  async storeApprovedStudies(studyName, studyAbbreviation) {
+      const approvedStudies = ApprovedStudies.createApprovedStudies(studyName, studyAbbreviation);
+      const res = await this.organizationCollection.findOneAndUpdate({studyName, studyAbbreviation}, approvedStudies);
```

Comment:
I think we should append new approved studies to existing **studies** instead of overwrite it.

Reply:
I've updated the logic. The unique approved study will be appended into the organization collection.